### PR TITLE
Refactor translations

### PR DESCRIPTION
**Pull request by paglias**
@Sinza- @luveluen

HabitRPG/habitrpg#2550

### CODE REVIEW

File: locales/en/main.json [hunk 1]
```diff
@@ -276,8 +274,7 @@
       "beastMastText" : "Has found all 90 pets (insanely difficult, give this user props!)",
       "beastMastUnlocked" : "You have earned the 'Beast Master' Achievement for collecting all the pets!",
     "rebirthBegan" : "Began a New Adventure",
-      "rebirthText1" : "Began ",
-      "rebirthText2" : " New Adventures",
+      "rebirthText" : "Began <%= rebirths %> New Adventures",
       "rebirthOrb" : "Used an Orb of Rebirth to start over after attaining Level ",
     "helped" : "Helped Habit Grow",
       "helpedText1" : "Helped HabitRPG grow by filling out ",
```
Comment (paglias) [inline locales/en/main.json:194]: I left out this one because it has a link in the middle and the second part of the string is the link's string

Comment (marcelosizer) [inline locales/en/main.json:194]: Ok, links out then.

Comment (paglias) [inline locales/en/main.json:194]: It's an idea... My goal is to make life the easier the possible for translators

**Comment by paglias** [comment]
@Sinza- if you want to split now :) I've removed all the unused strings

**Comment by paglias** [comment]
Some of the string aren't under the correct "_comment" some may have to be moved

File: locales/en/main.json [hunk 2]
```diff
@@ -542,7 +311,6 @@
 "characterbuild" : "Character Build",
 "class" : "Class",
 "warrior" : "Warrior",
-"wizard" : "Wizard",
```
Comment (marcelosizer) [inline locales/en/main.json:545]: Deleted wizard but not the others?

File: locales/en/main.json [hunk 3]
```diff
@@ -737,15 +505,12 @@
       "inviteParty" : "Invite Party",
       "questInvitation" : "Quest Invitation: ",
       "askLater" : "Ask Later",
-    "buyQuest" : "Buy Quest",
```
Comment (marcelosizer) [inline locales/en/main.json:740]: This line is for [habitrpg #2533](https://github.com/HabitRPG/habitrpg/pull/2533).

**Comment by marcelosizer** [comment]
No problem, I'll look for them and separate by files or folders.

File: locales/en/main.json (hunk 3 above)
Comment (paglias) [inline locales/en/main.json:740]: Ops, feel free to add it again (i can't until tomorrow), also scrollspre is used https://github.com/HabitRPG/habitrpg/pull/2533/files#diff-48c954af42797992266f6b5ad12d8ce3L50? I removed it too

File: locales/en/main.json (hunk 2 above)
Comment (paglias) [inline locales/en/main.json:545]: wizard I think is an old name, the others are used in the classes page

Comment (marcelosizer) [inline locales/en/main.json:545]: Wizard is still used at least on the avatar page. I'll switch that to Mage.

Comment (paglias) [inline locales/en/main.json:545]: Then it's probably not translated because I removed all the string that were never used

Comment (marcelosizer) [inline locales/en/main.json:545]: Ah, you're probably right!

File: locales/en/main.json (hunk 3 above)
Comment (marcelosizer) [inline locales/en/main.json:740]: Done.

**Pull request merged**